experimenter/experimenter/experiments/api/v5/serializers.py

```diff
@@ -295,6 +299,10 @@ def _get_feature_configs(self):
             .order_by("name")
         ]
 
+    def _get_population_sizing_data(self):
+        sizing_data = cache.get(SIZING_DATA_KEY)
+        return sizing_data.json() if sizing_data else "Sizing data not available."
```

Comment:
Oh I really gotta rip this thing out.  This serializer isn't used at all, it was added as part of an effort to migrate off of GQL and onto DRF but was never finished and this whole class is unused.